theme/mixins/backdrop.less

```diff
@@ -7,4 +7,5 @@
   background-color: @sk-black;
   opacity: 0.7;
   transition: opacity 0.3s;
+  z-index: @z-index-negative;
```

Comment:
Yeah I see the problem... so I guess another way to fix this is to have the consuming app provide a z-index value to indicate which box should sit over the other one.

I think using a negative z-index value for the backdrop will almost certainly cause a problem for someone using the backdrop at some point in the future.  If that happens I'm worried that the fix (which would involve removing the negative z-index) could accidentally re-introduce this bug and we're back where we started.  This bug breaks 2 apps that showcase owns and I'd really like for it to not break again